src/communication/communicator_impl.cpp

```diff
@@ -16,14 +16,14 @@ bool is_manageable(MPI_Comm mpi_communicator) noexcept {
 }
 
 CommunicatorImpl::CommunicatorImpl(MPI_Comm mpi_communicator) : comm_(mpi_communicator) {
-  DLAF_ASSERT(comm_ != MPI_COMM_NULL, "A null communicator is used!");
+  DLAF_ASSERT(comm_ != MPI_COMM_NULL, comm_);
```

Comment:
Not sure if this is useful... What can you deduce from the communicator id?

Reply:
You're right. I removed it.